series/series.go

```diff
@@ -818,3 +818,23 @@ func (s Series) Sum() float64 {
 	}
 	return sum
 }
+
+// Slice slices Series from j to k-1 index.
+func (s Series) Slice(j, k int) Series {
+	if err := s.Err; err != nil {
+		return s
+	}
+
+	if j > k {
```

Comment:
You should check `j >= 0` and `k<=s.Len()`.

Reply:
what do you mean by this? check `j >= 0` and `k < s.Len()` instead of the errors?

Reply:
I mean `j` have to be `>= 0` or returnn an error and `k` have to be `< s.Len()`.
Your commit hits what I wanted to say.